modules/motions/PlaCameraAction.h

```diff
@@ -30,6 +30,8 @@ class PlaCameraAction : public Motion {
    */
   void run() override;
 
+  void setUploadName(const std::string& fileName);
```

Comment:
説明のコメントがないね。あとuploadNameじゃなくてimageSaveNameとかが適切かな。アップロード画像のみを保存するわけじゃないから。
```suggestion
  /**
   * @brief 動体検知で得た画像のファイル名をセットする
   * @param 画像ファイル名
   */
  void setImageSaveName(const std::string& fileName);
```